src/dsp_tools/commands/ingest_xmlupload/bulk_ingest_client.py

```diff
@@ -70,3 +79,52 @@ def upload_file(
 
         logger.info(f"Uploaded file '{filepath}' to '{url}'")
         return None
+
+    def trigger_ingest_process(self) -> None:
+        """Start the ingest process on the server."""
+        url = f"{self.dsp_ingest_url}/projects/{self.shortcode}/bulk-ingest"
+        res = self.session.post(url, timeout=5)
+        if res.status_code in [STATUS_UNAUTHORIZED, STATUS_FORBIDDEN]:
+            raise BadCredentialsError("Unauthorized to start the ingest process. Please check your credentials.")
+        if res.status_code == STATUS_NOT_FOUND:
+            raise UserError(
+                f"No assets have been uploaded for project {self.shortcode}. "
+                "Before using the 'ingest-files' command, you must upload some files with the 'upload-files' command."
+            )
+        if res.status_code == STATUS_CONFLICT:
+            msg = f"Ingest process on the server {self.dsp_ingest_url} is already running. Wait until it completes..."
+            print(msg)
+            logger.info(msg)
+            return
+        if res.status_code in [STATUS_INTERNAL_SERVER_ERROR, STATUS_SERVER_UNAVAILABLE]:
+            raise UserError("Server is unavailable. Please try again later.")
+        try:
+            returned_shortcode = res.json().get("id")
+            failed: bool = returned_shortcode != self.shortcode
+        except JSONDecodeError:
+            failed = True
+        if failed:
+            raise UserError("Failed to trigger the ingest process. Please check the server logs, or try again later.")
+        print(f"Kicked off the ingest process on the server {self.dsp_ingest_url}. Wait until it completes...")
```

Comment:
This is cosmetic, but could you put this string in a variable since it is identical with the one in the line below?

Reply:
My habit is to do it this way as long as the string fits 1 line. Because then, I need 2 LOC over all (as opposed to 3 LOC if I put it into a variable). 

As soon as the message is 2 lines long, I put it into a variable, which results in less LOC.